python/lsst/sims/maf/bundles/__init__.py

```diff
@@ -0,0 +1 @@
+from .glanceBundle import *
```

Comment:
I'll make a ticket to do this--we should also decide if my simple column dictionary mapping is the way to go and incorporate that at the same time.